src/csi/pkg/pos/controllerserver.go

```diff
@@ -84,13 +202,13 @@ func (s *controllerServer) CreateVolume(
 		return nil, status.Error(codes.InvalidArgument, "Volume Capabilities missing in request")
 	}
 
-	klog.Infof("POS Volumes, %v", s.volumes)
-	posVolume, exists := s.volumes[req.Name]
+	klog.Infof("POS Volumes, %v", s.volumesByName)
+	posVolume, exists := s.volumesByName[req.Name]
 	klog.Infof("isExists %v %v", posVolume, exists)
 	if !exists {
 		posVolume = &volume{}
 		posVolume.status = CREATING
-		s.volumes[req.Name] = posVolume
+		s.volumesByName[req.Name] = posVolume
 	}
 	if exists {
```

Comment:
else can be use here.

Reply:
Good suggestion , I will change it 
Thanks